sympy/assumptions/facts.py

```diff
@@ -34,71 +36,138 @@ def get_composite_predicates():
 
 @cacheit
 def get_known_facts():
+    """
+    Facts between unary predicates.
+
+    Returns
+    =======
+
+    x : Placeholder symbol for unary facts.
+
+    fact : Known facts in conjugated normal form.
+
+    """
     # We build the facts starting with primitive predicates.
     # DO NOT include the predicates in get_composite_predicates()'s keys here!
-    return And(
-
+    x = Symbol('x')
+    fact = And(
         # primitive predicates exclude each other
-        Implies(Q.negative_infinite, ~Q.positive_infinite),
-        Implies(Q.negative, ~Q.zero & ~Q.positive),
-        Implies(Q.positive, ~Q.zero),
+        Implies(Q.negative_infinite(x), ~Q.positive_infinite(x)),
+        Implies(Q.negative(x), ~Q.zero(x) & ~Q.positive(x)),
+        Implies(Q.positive(x), ~Q.zero(x)),
 
         # build real line and complex plane
-        Implies(Q.negative | Q.zero | Q.positive, ~Q.imaginary),
-        Implies(Q.negative | Q.zero | Q.positive | Q.imaginary, Q.algebraic | Q.transcendental),
+        Implies(Q.negative(x) | Q.zero(x) | Q.positive(x), ~Q.imaginary(x)),
+        Implies(Q.negative(x) | Q.zero(x) | Q.positive(x) | Q.imaginary(x),
+                Q.algebraic(x) | Q.transcendental(x)),
 
         # other subsets of complex
-        Implies(Q.transcendental, ~Q.algebraic),
-        Implies(Q.irrational, ~Q.rational),
-        Equivalent(Q.rational | Q.irrational, Q.negative | Q.zero | Q.positive),
-        Implies(Q.rational, Q.algebraic),
+        Implies(Q.transcendental(x), ~Q.algebraic(x)),
+        Implies(Q.irrational(x), ~Q.rational(x)),
+        Equivalent(Q.rational(x) | Q.irrational(x),
+                Q.negative(x) | Q.zero(x) | Q.positive(x)),
```

Comment:
I think it would be useful to be able to use the composite predicates in these facts like:
```python
Equivalent(Q.real(x), Q.rational(x) | Q.irrational(x))
```